# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/css/css-masking/clip-path/clip-path-geometryBox-2.html.ini
# Commit: a6ec6ef3c92a
# Full Hash: a6ec6ef3c92a64edbd6534753c0deca9d13658d1
# Author: Jeff Muizelaar <jmuizelaar@mozilla.com>
# Date: 2022-02-24 03:51:41
# Regressor Bug: 1756197
# File Overlap Count: 1
# Description:
#   Bug 1756197. Avoid clipping if we're just going to fill the path. r=gfx-reviewers,nical
#   
#   This avoids having to rasterize the path twice. Once for the clip path and once for
#   the fill. It will also give us correct antialiasing on the edges.
#   
# ==============================================================================

diff -r 13a75f8c612b -r a6ec6ef3c92a testing/web-platform/meta/css/css-masking/clip-path/clip-path-geometryBox-2.html.ini
--- a/testing/web-platform/meta/css/css-masking/clip-path/clip-path-geometryBox-2.html.ini	Wed Feb 23 16:19:07 2022 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,2 +0,0 @@
-[clip-path-geometryBox-2.html]
-  expected: FAIL
